dinosure_course_gordon_fleming/.root-auth

```diff
@@ -0,0 +1 @@
+ROOT_API_KEY=sandbox_ODgwNDU1Y2YtYzJlYy00MTFmLTkzNDItNTIxMTU1MTBmYTE2LlQ4ME92TmxZVFhOeTZWckZsQk9zTUJTSmhLZzBmNUVu
```

Comment:
Needs to be part of `.gitignore` - very important not to have API keys stored in repos. Please remove and recommit so that its stored on your local and ignored by git.